varats/varats/tools/tool_util.py

```diff
@@ -64,3 +66,22 @@ def get_research_tool(
 def get_supported_research_tool_names() -> tp.List[str]:
     """Returns a list of all supported research tools."""
     return ["phasar", "vara"]
+
+
+def configuration_lookup_error_handler(func: tp.Any) -> tp.Any:
+    """Wrapper for drivers to catche internal Exceptions and provide a helpful
+    message to the user."""
+
+    @wraps(func)
+    def wrapper_configuration_lookup_error_handler(
+        *args: tp.Any, **kwargs: tp.Any
+    ) -> tp.Any:
```

Comment:
I think, we should restrict the return type to be a function:
```suggestion
    ) -> tp.Callable[..., tp.Any]:
```